solidity/contracts/converter/ConverterBase.sol

```diff
@@ -446,6 +432,29 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
         emit Conversion(_sourceToken, _targetToken, _trader, _amount, _returnAmount, int256(_feeAmount));
     }
 
+    /**
+     * @dev transfers funds held by the contract and sends them to an account
+     *
+     * @param token ERC20 token contract address
+     * @param to account to receive the new amount
+     * @param amount amount to withdraw
```

Comment:
You have a redundant 'amount' here ('amount amount')

Reply:
No, `amount` is the name of the parameter